includes/Classifai/Providers/OpenAI/ChatGPT.php

```diff
@@ -423,7 +473,7 @@ public function generate_excerpt( int $post_id = 0 ) {
 
 		// These checks (and the one above) happen in the REST permission_callback,
 		// but we run them again here in case this method is called directly.
-		if ( empty( $settings ) || ( isset( $settings['authenticated'] ) && false === $settings['authenticated'] ) || ( isset( $settings['enable_excerpt'] ) && 'no' === $settings['enable_excerpt'] ) ) {
+		if ( ! $this->user_can_access( 'enable_excerpt' ) ) {
```

Comment:
As mentioned above, if we remove auth checks and enabled checks from the `user_can_access` method, I think these lines can stay as they are

Reply:
@dkotter, The fix for this is now ready for review. 

cc @jeffpaul